tools/gfx/vk-module.h

```diff
@@ -12,7 +12,7 @@
 #endif
 
 #define VK_NO_PROTOTYPES
-#include <vulkan/vulkan.h>
+#include <vulkan/include/vulkan/vulkan.h>
```

Comment:
This isn't beautiful, which is probably why I went and manually created an "unpacked" layout for things previously, but I'm fine with it if this will make it easier to upgrade our VK header version in the future.